.travis.yml

```diff
@@ -8,21 +8,21 @@ install:
   - "bundle exec rake neo4j_spatial:install"
   - "bundle exec rake neo4j:start --trace"
   - "sleep 20"
+  - "bundle exec rake neo4j:generate_schema_migration[constraint,Restaurant,uuid]"
```

Comment:
You can certainly do this, but the specs won't work on other peoples' machines unless they know to run this line.    I generally try to do something like this inside the specs themselves:

```ruby
Neo4j::ActiveBase.label_object(: Restaurant).create_constraint(:uuid, type: :unique)
```

You might also rescue from `Neo4j::Core::CypherSession::SchemaErrors::IndexAlreadyExistsError` so that if it already exists it doesn't just fail